### ISSUE TITLE
Dotnet publish does not work when DACPAC output name is changed

### ISSUE THREAD

**Issue by RVink** [rr-wfm/MSBuild.Sdk.SqlProj#68]
As described in #65 it's possible to change the name of the DACPAC output by setting the `AssemblyName` in the project file. However changing the output name breaks the `dotnet publish` command. This is because the publish command tries to use the DACPAC file based on the project name.

`C:\Users\rvink\.nuget\packages\msbuild.sdk.sqlproj\1.7.0\Sdk\Sdk.targets(199,5): error MSB3073: The command "dotnet "C:\Users\rvink\.nuget\packages\msbuild.sdk.sqlproj\1.7.0\Sdk\../tools/netcoreapp3.1/DacpacTool.dll" deploy -i "C:\Users\rvink\source\repos\TestProject\TestDatabase\bin\Debug\netstandard2.0\/TestDatabase.dacpac" -tsn "(local)" -tdn "RandomDatabaseName"   -p CreateNewDatabase=False -p IncludeCompositeObjects=true  " exited with code 1. [C:\Users\rvink\source\repos\TestProject\TestDatabase\TestDatabase.csproj]`

This is my project file:
```
<Project Sdk="MSBuild.Sdk.SqlProj/1.7.0">
    <PropertyGroup>
        <TargetFramework>netstandard2.0</TargetFramework>
        <SqlServerVersion>Sql140</SqlServerVersion>
	<AssemblyName>RandomDatabaseName</AssemblyName>
	<TargetDatabaseName>RandomDatabaseName</TargetDatabaseName>
        <!-- For additional properties that can be set here, please refer to https://github.com/rr-wfm/MSBuild.Sdk.SqlProj#model-properties -->
    </PropertyGroup>

    <PropertyGroup>
        <!-- Refer to https://github.com/rr-wfm/MSBuild.Sdk.SqlProj#publishing-support for supported publishing options -->
    </PropertyGroup>
</Project>
```

**Comment by brettrowberry** [comment]
I just observed the same thing. I guess I tried building but not publishing when I added `AssemblyName`. 

I'm documenting my thoughts here.

`dotnet build` works when the following are defined:
- `AssemblyName`
- `TargetDatabaseName`
- `AssemblyName` and `TargetDatabaseName`
- neither is defined

`dotnet publish` works when the following are defined:
- `TargetDatabaseName`
- neither is defined

`dotnet publish` does not work when the following are defined:
- `AssemblyName`
- `AssemblyName` and `TargetDatabaseName`
